examples/gno.land/r/nt/commondao/render.gno

```diff
@@ -1,6 +1,523 @@
+		ufmt.Sprintf("[List Proposals](%s)", daoProposalsURL(dao)),
+		ufmt.Sprintf("[View Settings](%s)", settingsURL(dao)),
+	}
+
+	// TODO: Add submenu with proposal links when proposals support is enabled
+
+	if parentDAO := dao.Parent(); parentDAO != nil {
+		menu = append(menu, ufmt.Sprintf("[Go to Parent DAO](%s)", daoURL(parentDAO)))
+	}
+
+	res.Write(strings.Join(menu, " • "))
+	res.Write("\n\n---\n")
+
+	// Render members
+	members := dao.Members()
+	if members.Size() == 0 {
+		res.Write("**⚠ The DAO has no members**\n\n")
+	} else {
+		renderMembers(res, req.RawPath, members)
+	}
+
+	// Render organization tree
+	if dao.Children().Len() > 0 {
+		r := parseRealmPath(req.RawPath)
+		dismissedVisible := r.Query.Has("dismissed") || dao.IsDeleted()
+		if dismissedVisible || !dao.IsDeleted() {
+			res.Write("## Tree\n")
+
+			// Render toggle only when DAO is not dismissed,
+			// otherwise render the whole tree when DAO is dismissed
+			if !dao.IsDeleted() {
+				res.Write("Dismissed: ")
+
+				if dismissedVisible {
+					r.Query.Del("dismissed")
+					res.Write(ufmt.Sprintf("[hide](%s)", r.String()))
+				} else {
+					r.Query.Add("dismissed", "")
+					res.Write(ufmt.Sprintf("[show](%s)", r.String()))
+				}
+				res.Write("\n\n")
+			}
+
+			renderTree(res, dao, "", dismissedVisible)
+		}
+	}
+
+	// Render latest proposals
+	proposals := dao.ActiveProposals()
+	if proposals.Size() == 0 {
+		return
+	}
+
+	res.Write("## Latest Proposals\n")
+	proposals.Iterate(0, 3, true, func(p *commondao.Proposal) bool {
+		renderProposalsListItem(res, dao, p)
+		return false
+	})
+}
+
+func renderMembers(res *mux.ResponseWriter, path string, members commondao.MemberStorage) {
+	pages, err := pager.New(path, members.Size(), pager.WithPageQueryParam("members"), pager.WithPageSize(8))
+	if err != nil {
+		res.Write(err.Error())
+		return
+	}
+
+	res.Write("| Members |\n| --- |\n")
+	members.IterateByOffset(pages.Offset(), pages.PageSize(), func(addr std.Address) bool {
+		res.Write("| ")
+		res.Write(userLink(addr))
+		res.Write(" |\n")
+		return false
+	})
+
+	if pages.HasPages() {
+		res.Write("\n")
+		res.Write(pager.Picker(pages))
+		res.Write("\n")
+	}
+
+	res.Write("\n")
+}
+
+func renderTree(res *mux.ResponseWriter, dao *commondao.CommonDAO, indent string, showDismissed bool) {
+	if dao.IsDeleted() {
+		// Strikethough dismissed DAO names and add a help tooltip
+		res.Write(ufmt.Sprintf("%s- [~~%s~~](%s 'Dismissed DAO')\n", indent, dao.Name(), daoURL(dao)))
+	} else {
+		res.Write(ufmt.Sprintf("%s- [%s](%s)\n", indent, dao.Name(), daoURL(dao)))
+	}
+
+	indent += "  "
+	dao.Children().ForEach(func(_ int, v any) bool {
+		subDAO, ok := v.(*commondao.CommonDAO)
+		if !ok {
+			return false
+		}
+
+		if showDismissed || !subDAO.IsDeleted() {
+			renderTree(res, subDAO, indent, showDismissed)
+		}
+		return false
+	})
+}
+
+func renderSettings(res *mux.ResponseWriter, req *mux.Request) {
+	dao := mustGetDAOFromRequest(req)
+	o := getOptions(dao.ID())
```

Comment:
Options are settings for dApps that choose to use `commondao` realm to create their DAOs. CommonDAO realm allows voting and execution of proposals which are opt in features. Also SubDAO support is an opt in feature for hierarchical DAOs.